examples/Images and Tables.ipynb

```diff
@@ -0,0 +1,397 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following line is needed to load the Firefly Javascript API into the DOM. "
```

Comment:
This notebook cell is no longer applicable and should be removed.

Reply:
fixed